app/front_end/OTTOfficeTimeTracker/app/src/main/java/com/example/officetimetracker/ProfileActivity.java

```diff
@@ -2,13 +2,28 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
 
 public class ProfileActivity extends AppCompatActivity {
 
+    Button signOut;
```

Comment:
I would keep a sigOut button in the main menu also and add here a 'save' button for profile update.